### ISSUE TITLE
Migration error RemoveMailChimpBundleAndAbandonedCartBundleConfigs

### ISSUE THREAD

**Issue by sadortun** [oroinc/crm#333]
Hi Oro Team,

While migrating from 2.6 to 3.1, the migration in `src\Oro\Bundle\CRMBundle\Migrations\Schema\v1_7\RemoveMailChimpBundleAndAbandonedCartBundleConfigs.php` cause errors since none of theses columns exists

```
            $queries->addQuery('ALTER TABLE oro_integration_transport DROP orocrm_mailchimp_apikey;');
            $queries->addQuery('ALTER TABLE oro_integration_transport DROP orocrm_mailchimp_act_up_int;');
            if ($this->platform instanceof PostgreSqlPlatform) {
                $queries->addQuery('ALTER TABLE orocrm_cmpgn_transport_stngs DROP CONSTRAINT FK_16E86BF27BC28329;');
                $queries->addQuery('ALTER TABLE orocrm_cmpgn_transport_stngs DROP CONSTRAINT FK_16E86BF27162EA00;');
                $queries->addQuery('DROP INDEX IDX_16E86BF27162EA00;');
                $queries->addQuery('DROP INDEX IDX_16E86BF27BC28329;');
            } else {
                $queries->addQuery('ALTER TABLE orocrm_cmpgn_transport_stngs DROP FOREIGN KEY FK_16E86BF27BC28329;');
                $queries->addQuery('ALTER TABLE orocrm_cmpgn_transport_stngs DROP FOREIGN KEY FK_16E86BF27162EA00;');
                $queries->addQuery('ALTER TABLE orocrm_cmpgn_transport_stngs DROP INDEX IDX_16E86BF27BC28329;');
                $queries->addQuery('ALTER TABLE orocrm_cmpgn_transport_stngs DROP INDEX IDX_16E86BF27162EA00;');
            }
            $queries->addQuery('ALTER TABLE orocrm_cmpgn_transport_stngs DROP mailchimp_template_id;');
            $queries->addQuery('ALTER TABLE orocrm_cmpgn_transport_stngs DROP mailchimp_channel_id;');
            $queries->addQuery('ALTER TABLE orocrm_cmpgn_transport_stngs DROP mailchimp_receive_activities;');
```

Have a good day,
Samuel

**Comment by mbessolov** [comment]
Thank you for the report.
We'll check what could go wrong.

_Internal reference: CRM-9135_

**Comment by anyt** [comment]
The bug has been fixed in 3.1.6. Thank you!
